package.json

```diff
@@ -117,7 +117,18 @@
                     "application/vnd.vegalite.v4+json",
                     "application/vnd.vegalite.v5+json"
                 ]
+            },
+            {
+                "id": "jupyter-markdown",
+                "displayName": "Jupyter Markdown styles",
+                "entrypoint": {
+                    "extends": "vscode.markdown-it-renderer",
+                    "path": "./out/client_renderer/markdown.js"
+                }
             }
+        ],
+        "markdown.previewStyles": [
+            "resources/jupyter-markdown.css"
```

Comment:
This will only apply inside the markdown preview. Is that what you want?

Reply:
Ok, from the issue deception though it sounds like this is specific to notebooks? If so, then you don't need the `markdown.previewStyles` part

Reply:
works for me